### PR TITLE
Autogenerated "Authors' addresses: ..." and authors with multiple affiliations

### PR DESCRIPTION
**Pull request by sommer**
If I understood the current version of the guide document right, an author with multiple affiliations is supposed to be entered as...
```
\author{a}
\affiliation{\institution{i1}\city{c1}}
\affiliation{\institution{i2}\city{c2}}
```
For journals, this produces an "Authors' addresses: " line that reads something like...
```
a, i1, c1, i2, c2;
```
...which could be hard to parse visually.

Aside from asking an author to manually specify the complete `\authorsaddresses` line there is probably no easy solution for complex cases - but maybe simply adding an "`and`" in-between multiple institutions of a single author...
```
a, i1, c1 and i2, c2;
```
...can take autogeneration a little step further.

I am attaching a (highly simplistic) proof of concept patch.

### CODE REVIEW

**Comment by borisveytsman** [comment]
Thanks -an interesting idea.  @craig-rodkin, what do you think?

**Comment by Craig-Rodkin** [comment]
Hi Boris,

I would need to pass this by editorial. Can you or Sommer provide a PDF sample of current and proposed output for me to pass along to staff?

Be well.
Regards,

Craig

Craig Rodkin
ACM Publications Operations Manager
***@***.******@***.***>

**Comment by sommer** [comment]
Sure! I'm attaching the result of adding a second affiliation to the first two authors of `sample-acmsmall.tex` which comes with `acmart` (before and after the proposed change).

[before.pdf](https://github.com/borisveytsman/acmart/files/6300076/before.pdf) vs. [after.pdf](https://github.com/borisveytsman/acmart/files/6300077/after.pdf)

For convenience, here's a side-by-side of the relevant part of the first page.

sidebyside

**Comment by Craig-Rodkin** [comment]
Thanks Christoph,

I’m sending your request to the editorial team for review. I’ll let you know their decision as soon as possible.

Be well.
Regards,

Craig

Craig Rodkin
ACM Publications Operations Manager
***@***.******@***.***>
